packaging/rpm/cvmfs-universal.spec

```diff
@@ -10,6 +10,8 @@
 %endif
 %if 0%{?el7} || 0%{?fedora}
 %define selinux_cvmfs_server 1
+%endif
+%if 0%{?el7} || 0%{?fc29}
```

Comment:
Is there a possibility to check for fedora >= 29?

Reply:
On https://github.com/cvmfs/cvmfs/pull/2323/commits/7b6f09c62809de6883592b1defea63ea47105085